packages/connect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@substrate/connect",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
Even though 0 version does not matter we should get used to following this:
0. | MAJOR version when you make incompatible API changes,
0. | MINOR version when you add functionality in a backwards compatible manner, and
0 | PATCH version when you make backwards compatible bug fixes. <----

Reply:
This is not backwards compatible.  The interface of the message sent between the extension and @substrate/connect has changed.  It is a backwards incompatible fix.

Reply:
hmmm actually I see what you mean ... the **public** API has not changed .. I.E. the interface of `Detector` so it should be a revision change.  i'll change it to 0.3.1

Reply:
this is interesting .. we need to think give some thought about the implicit interface of messages passed between the `ExtensionProvider` in @substrate/connect and the `ExtensionMessageRouter` in the extension.  When we go public with this we'll basically be locking ourself into that interface because changing it will be **very** hard. (actually impossible with the current design)